### ISSUE TITLE
How to Identify the pressed BarChart bar when onPress is triggered.

### ISSUE THREAD

**Issue by 03balogun** [Abhinandan-Kushwaha/react-native-gifted-charts#159]
Hello,

Great work on the library.

Is there a way to identify the clicked/pressed bar when onPress is triggered?

Currently, it returns the class properties of the Pressed bar. The bar index or data doesn't  return with the callback.

### Code Example:

```
const App = () => {
  const onBarPressed = (params, index, data) => {
    // I want to be able to do
    console.log(params.index);
    console.log(params.data);

    // Or this
    console.log(index);
    console.log(data);
  };

  const barData = [
    { value: 0.7, label: '1', onPress: onBarPressed },
    { value: 0.8, label: '2', onPress: onBarPressed },
    { value: 0.6, label: '3', onPress: onBarPressed },
    { value: 0.4, label: '4', onPress: onBarPressed },
    { value: 0.9, label: '5', onPress: onBarPressed },
    { value: 0.7, label: '6', onPress: onBarPressed }
  ];
  return (
    <View>
      <BarChart
        showFractionalValues
        showYAxisIndices
        showXAxisIndices
        hideRules
        noOfSections={5}
        data={barData}
        showGradient
        frontColor={'#1B6BB0'}
        gradientColor={'#FFEEFE'}
        backgroundColor={'#FECF9E'}
      />
    </View>
  );
};
````

I will happy to create a PR for this update, if a way to do this doesn't currently exists.

**Comment by Abhinandan-Kushwaha** [comment]
Hi @03balogun 👋
Thanks for requesting the feature. It is now available in version `1.2.7`🎉

Now, you can directly pass `onPress` as a prop and it will accept 2 parameters- `item` and `index`

So your new code will be-
```js
const App = () => {
  const onBarPressed = (item, index) => {
    console.log(item);
    console.log(index);
  };

  const barData = [
    { value: 0.7, label: '1'},
    { value: 0.8, label: '2'},
    { value: 0.6, label: '3'},
    { value: 0.4, label: '4'},
    { value: 0.9, label: '5'},
    { value: 0.7, label: '6'}
  ];
  return (
    <View>
      <BarChart
        showFractionalValues
        showYAxisIndices
        showXAxisIndices
        hideRules
        noOfSections={5}
        data={barData}
        showGradient
        frontColor={'#1B6BB0'}
        gradientColor={'#FFEEFE'}
        backgroundColor={'#FECF9E'}
        onPress={onBarPressed}
      />
    </View>
  );
};
```
I hope it helps.🤗

**Comment by 03balogun** [comment]
Thanks @Abhinandan-Kushwaha this works 💪🏼
